frontend/src/app/modules/user-profile/select-topics-page/select-topics-page.component.sass

```diff
@@ -0,0 +1,112 @@
+@import '../../../../styles/colors'
+
+@font-face
+    font-family: Barlow
+    src: url('../../../../assets/fonts/Barlow-Medium.ttf')
+
+main
+    width: 50%
+    margin: 30px auto 0 auto
+    font-family: Barlow, serif
+
+#back
+    background-color: $white
+    padding: 10px
+    border: none
+    width: 15%
+    display: flex
+    justify-content: center
+    align-items: center
+
+#back:hover, button:hover
+    background-color: $gray-200
+    cursor: pointer
+    transition: background-color 0.3s ease-out
+
+#back img
+    padding: 0 10px 0 0
+
+h2
+    color: $main-color
+    margin: 10px 0 0 0
+
+#topics
+    display: grid
+    grid-template-columns: repeat(5, 1fr)
+    grid-gap: 15px
+    height: 100%
+    margin: 30px 0
+
+.topic
+    height: 100%
+    width: 100%
+    display: flex
+    flex-direction: column
+    justify-content: center
+    align-items: center
+
+.topic img
+    height: 65%
+
+.not-selected
+    color: $gray-300
+
+.not-selected div
+    background-color: $inactive-color
+
+.selected
+    color: $black
+
+.selected div
+    background-color: $time-color
```

Comment:
time-color is not a good choise

Reply:
But this are the colors from figma, you want me to rename variables or colors?

Reply:
rename variables